dev_guide/getting_traffic_into_cluster.adoc

```diff
@@ -198,37 +237,41 @@ xref:using-ingress-IP-self-service[Ingress IP Self-Service] provides a solution.
 Ingress IP Self-Service streamlines the allocation of External IPs for accessing
 services in the cluster. Cluster administrators can designate a range of
```

Comment:
Add a xref to "designate a range of addresses" ?
xref:../install_config/master_node_configuration.adoc#master-node-config-network-config